examples/rag_on_wikipedia_article/retrieve_documents.py

```diff
@@ -0,0 +1,32 @@
+"""Retrieve documents from the Chroma vector store."""
+import os
+
+from rag_config import settings
+from stores.wikipedia_chroma import WikipediaStore
+
+from mirascope.anthropic import AnthropicCall
+
+if settings.anthropic_api_key:
+    os.environ["ANTHROPIC_API_KEY"] = settings.anthropic_api_key
+if settings.openai_api_key:
+    os.environ["OPENAI_API_KEY"] = settings.openai_api_key
+
+
+class WikipediaCall(AnthropicCall):
+    prompt_template = """
+    SYSTEM: Answer the question based on the context.
```

Comment:
for multiline prompt strings, always start on the next line e.g.
```python
SYSTEM:
Answer the question based on the context:
{context}
```

This will ensure `dedent` works properly.

Reply:
done